integration-tests/src/test/resources/indexer/wikipedia_index_queries_with_transform.json

```diff
@@ -0,0 +1,54 @@
+[
+    {
+        "description":"having spec on post aggregation",
+        "query":{
+            "queryType":"groupBy",
+            "dataSource":"%%DATASOURCE%%",
+            "granularity":"day",
+            "dimensions":[
+                "page"
```

Comment:
Using a filter on "newPage" fails the test locally. But if I apply the transformation from another datasource it appears to work. Need to verify if this is actually a bug or not

Reply:
This is a bug filed #9593 for this